### PR TITLE
Add more content

### PR DESCRIPTION
**Pull request by subzero79**

### CODE REVIEW

File: FAQ.rst [hunk 1]
```diff
@@ -43,8 +43,13 @@ Why does the system rewrites a configuration file(s) that I have manually edited
 How can I modify an internal value of some service |omv| has control over?
 	Read :doc:`here <various/advancedsettings>` for advanced configurations.
 
+<<<<<<< HEAD
```
Comment (votdev) [inline FAQ.rst]: Are these changes on purpose? Maybe you should update your repo, i had to rebase it some time ago to get rid of useless commits. I know, bad idea on a public repo, but for the moment there is less activity.

Comment (subzero79) [reply FAQ.rst]: I had a conflict when merging from upstream for this file. I though I solved the conflict but it wasn’t.
